packages/client/hmi-client/src/components/workflow/ops/funman/tera-funman.vue

```diff
@@ -316,6 +313,15 @@ const toggleAdditonalOptions = () => {
 	showAdditionalOptions.value = !showAdditionalOptions.value;
 };
 
+const menuItems = computed(() => [
+	{
+		label: 'Save as new model',
+		icon: 'pi pi-pencil',
```

Comment:
Funman doesn't produce anything right now, but eventually it will produce model-configurations, not models.